README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# hw01
+
+# Who am I?
+
+![](https://sociology.uchicago.edu/sites/sociology.uchicago.edu/files/styles/columnwidth-wider/public/uploads/images/Betsy%20Priem%20photo.jpg?itok=CiRQvArA)
```

Comment:
This is a URL, so it's a universal link. Make sure you can also use a relative path to insert images -- it might be useful down the road!